scripts/deployment/phase2-assets/collaterals/deploy_origin_oeth.ts

```diff
@@ -0,0 +1,93 @@
+import fs from 'fs'
+import hre from 'hardhat'
+import { getChainId } from '../../../../common/blockchain-utils'
+import { baseL2Chains, networkConfig } from '../../../../common/configuration'
+import { bn, fp } from '../../../../common/numbers'
+import { expect } from 'chai'
+import { CollateralStatus } from '../../../../common/constants'
+import {
+  PRICE_FEEDS,
+  ORACLE_ERROR,
+  OETH_ORACLE_ERROR,
+  ORACLE_TIMEOUT,
+  OETH_ORACLE_TIMEOUT,
+} from '../../../../test/plugins/individual-collateral/origin/constants'
+import {
+  getDeploymentFile,
+  getAssetCollDeploymentFilename,
+  IAssetCollDeployments,
+  getDeploymentFilename,
+  fileExists,
+} from '../../common'
+import { priceTimeout } from '../../utils'
+import { OETHCollateral } from '../../../../typechain'
+import { ContractFactory } from 'ethers'
+
+async function main() {
+  // ==== Read Configuration ====
+  const [deployer] = await hre.ethers.getSigners()
+
+  const chainId = await getChainId(hre)
+
+  console.log(`Deploying Origin ETH to network ${hre.network.name} (${chainId})
+    with burner account: ${deployer.address}`)
+
+  if (!networkConfig[chainId]) {
+    throw new Error(`Missing network configuration for ${hre.network.name}`)
+  }
+
+  // Get phase1 deployment
+  const phase1File = getDeploymentFilename(chainId)
+  if (!fileExists(phase1File)) {
+    throw new Error(`${phase1File} doesn't exist yet. Run phase 1`)
+  }
+  // Check previous step completed
+  const assetCollDeploymentFilename = getAssetCollDeploymentFilename(chainId)
+  const assetCollDeployments = <IAssetCollDeployments>getDeploymentFile(assetCollDeploymentFilename)
+
+  const deployedCollateral: string[] = []
+
+  /********  Deploy Origin ETH Collateral - wOETH  **************************/
+  const OETHCollateralFactory: ContractFactory = await hre.ethers.getContractFactory(
+    'OETHCollateral'
+  )
+
+  const collateral = <OETHCollateral>await OETHCollateralFactory.connect(
+    deployer
+  ).deploy(
+    {
+      priceTimeout: priceTimeout.toString(),
+      chainlinkFeed: PRICE_FEEDS.OETH_ETH, // ETH/OETH
+      oracleError: (ORACLE_ERROR.add(OETH_ORACLE_ERROR)).toString(),
```

Comment:
i would use the `combinedError` helper to match the verification script and all other deployment scripts.

Reply:
Indeed! Done in this commit [eaa8b4e](https://github.com/reserve-protocol/protocol/pull/1239/commits/eaa8b4e622294558811e5fd942def655d0d31f54)